### DISCUSSION TITLE
Why doesn't work events?

### DISCUSSION THREAD

**Discussion by m3r1v3** [Rapptz/discord.py#6490] [Q&A] [answered]
I made my bot a lot time ago. Before my events work is good, but now they doesn't work. Before I used _v1.4.*_. I read somewhere that this system was changed in _v1.5_. What exactly can be the problem. Example, my `on_member_join/on_member_remove` doesn't work, but `on_ready/on_message` work right.

**Comment by Zomatree** [comment, marked as answer]
you need to enable the `members` intent to use `on_member_join/remove`, read this for details on how to enable them https://discordpy.readthedocs.io/en/latest/intents.html

**Reply by m3r1v3** [reply]
Thanks
